### ISSUE TITLE
Replacing Arduinos with Jetson Tx2

### ISSUE THREAD

**Issue by Rafcin** [XRobots/openDog#8]
I'm just pondering some ideas, but could you instead run the whole robot from a Jetson Tx2 and have ROS so then you can add other sensors and having multiple programs in different languages.

**Comment by XRobots** [comment]
Yes probably, there are lots of options. A fair way to go before upgrades yet though!

**Comment by zbrown23** [comment]
https://github.com/FRC900/2018RobotCode 
this is a FIRST Robotics team's code, but they are one of if not the only team to use ROS for their competition robot. It's way over my head but maybe you guys can make sense of it and possibly use it. I'd love to see XRobots projects running ROS.
